CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @digitalbazaar/ed25519-signature-2020 Changelog
 
+## 6.0.0 - 2023-TBD
```

Comment:
We actually already removed support for `expansionMap` in version 5.0 by requiring the use of `jsonld-signatures@11`, so we don't need to list this as a breaking change. It's just clean up -- `expansionMap` already has no effect.

```suggestion
## 5.2.0 - 2023-TBD
```

Reply:
Updated [75c4354](https://github.com/digitalbazaar/ed25519-signature-2020/pull/26/commits/75c43540647f1461ae61e0e3d5277fd39dd7ca98)